src/core/lbgpu.cpp

```diff
@@ -440,4 +442,33 @@ int lb_lbfluid_load_checkpoint_wrapper(char* filename, int binary)
   return lb_lbfluid_load_checkpoint(filename, binary);
 }
 
+void lb_lbfluid_particles_add_momentum ( float velocity[3] )
```

Comment:
Sure, why not.  In the core there is not really a difference because everything is made dimensionless.